data/json/monsters/zanimal_upgrade.json

```
@@ -67,6 +67,91 @@
     ],
     "extend": { "flags": [ "GROUP_BASH", "PUSH_VEH" ] }
   },
+  {
+    "id": "mon_dog_cuspated",
+    "type": "MONSTER",
+    "name": { "str": "cuspated dog" },
+    "//": "Cuspated just means angled, sharp, pointy. Other ways of translating the feel of the name may be something akin to 'scythe dog', or 'beaked beast'.",
+    "description": "This dog's snout and jaw have seemingly fused together, the skin and flesh ranging from completely gone or barely hanging on.  The boney mass as a whole has an incredibly sharp yet jagged point.  It stumbles around, barely keeping its hefty 'beak' from hitting the ground.",
+    "default_faction": "zombie",
+    "bodytype": "dog",
+    "species": [ "ZOMBIE" ],
+    "volume": "30000 ml",
+    "weight": "40750 g",
+    "hp": 30,
+    "speed": 65,
+    "material": [ "flesh" ],
+    "symbol": "d",
+    "color": "white_red",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "stab", "amount": 10, "armor_penetration": 8 } ],
+    "grab_strength": 0,
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_zombie" ],
+    "vision_day": 15,
+    "vision_night": 5,
+    "harvest": "zombie_dog",
+    "//2": "Can't bite because it doesn't exactly have a mouth anymore. Instead, it's effectively a one-hit, one-trick wonder. It tries to drive its extremely sharp beak into you. If it succeeds, you're thoroughly damaged. If it fails, it's screwed.",
+    "special_attacks": [ { "id": "wicked_impale", "attack_upper": false } ],
+    "death_drops": "mon_dog_death_drops",
+    "fungalize_into": "mon_zombie_dog_fungus",
+    "flags": [ "SEES", "STUMBLES", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ],
+    "//3": "Minor amounts of protection due to the boney jaw.",
+    "armor": { "cut": 5, "stab": 8, "acid": 1, "bullet": 8, "electric": 1 }
+  },
+  {
+    "id": "mon_dog_howling",
+    "type": "MONSTER",
+    "name": { "str": "howling dog" },
+    "description": "The underbelly of this undead canine expands and contracts arrythmically, almost like it's breathing.  Despite that, it is clearly in an advanced state of decomposition.  Its snout and throat appear far too large for its species, and the remains of its eyes appear much more analytical than its brethren.",
```

```suggestion
    "description": "The underbelly of this undead canine expands and contracts arrhythmically, almost like it's breathing.  Despite that, it is clearly in an advanced state of decomposition.  Its snout and throat appear far too large for its species, and the remains of its eyes appear much more analytical than its brethren.",
```
I really hate that the double r is more acceptable but either way this is misspelled